### DISCUSSION TITLE
在侧栏 Sidebar上如何设置联系我大图标

### DISCUSSION THREAD

**Discussion by youngdajie** [MaLuns/hexo-theme-async#104] [General 大局为重]
看了大佬的配置文档，跟着差不多弄好了，但看到还有一个地方没对，就是我看到很多人侧栏上会有一个大按钮 **_联系我_**，点击就跳转到邮箱了。我在配置文档里没找到，可能是我没找到吧，还请各位小伙伴能教一下我

![image](https://github.com/MaLuns/hexo-theme-async/assets/79750210/77342f87-1cff-4afd-b0a8-c5ae0f728c4b)

**Comment by MaLuns** [comment]
在用户信息里配置好邮箱就会自动显示了

https://hexo-theme-async.imalun.com/guide/config.html#用户信息-user

**Reply by youngdajie** [reply]
原来这么简单，谢谢大佬，膜拜😊
